bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -35,16 +35,39 @@ spec:
           #!/bin/bash
           set -euo pipefail
           export SYSTEMD_IGNORE_CHROOT=yes
+          # systemctl may not be used in a separate PID namespace to reach
+          # the systemd running in PID 1. Therefore we need to use the dbus API
+          systemctl_is_active(){
+            gdbus call \
+              --system \
+              --dest org.freedesktop.systemd1 \
+              --object-path /org/freedesktop/systemd1/unit/"$(svc_encode_name ${1})"_2eservice \
+              --method org.freedesktop.DBus.Properties.Get "org.freedesktop.systemd1.Unit" "ActiveState"  |
+                fgrep "(<'active'>,)" -q
```

Comment:
this won't match the other possible states: `reloading, inactive, failed, activating, deactivating`

There was a thread somewhere on slack about this.  I think we really only care if the systemd unit is "enabled", but "enabled" is too hard to track.  Even if the unit was started correctly on boot but has since restarted or failed we still want to know that it is supposed to be running on the host and not in the container.

`systemctl status ovsdb-server` exit status is blunt indicator that `ovsdb-server` is running or restarted, etc.

I think a better check would be to check for `ActiveEnterTimestamp > (<uint64 0>,) `  because that indicates that at some point since boot `ovsdb-server` was running, so it is our intention to run `ovsdb-server` on the host and not in a container, and it does not depend on whatever the current state of `ovsdb-server` is, be it `activating` or `reloading`, etc.

cc @trozet